sr_multi_moveit/sr_multi_moveit_config/launch/trajectory_execution.launch.xml

```diff
@@ -13,7 +13,7 @@
   <!-- Allow more than the expected execution time before triggering a trajectory cancel (applied after scaling) -->
   <param name="trajectory_execution/allowed_goal_duration_margin" value="0.5"/> <!-- default 0.5 -->
   <!-- Allowed joint-value tolerance for validation that trajectory's first point matches current robot state -->
-  <param name="trajectory_execution/allowed_start_tolerance" value="0.03"/> <!-- default 0.01 -->
+  <param name="trajectory_execution/allowed_start_tolerance" value="6.0"/> <!-- default 0.01 -->
```

Comment:
What function from the commander were you using to generate the plan/trajectory?
It seems that moveit checks the trajectory before sending it to the trajectory controller and it expects the first point to be in the current position.